sentinel-frontend/app/error.tsx

```
@@ -0,0 +1,77 @@
+'use client';
+
+import { useEffect } from 'react';
+import { RefreshCw, Home, AlertTriangle } from 'lucide-react';
+import Link from 'next/link';
+
+interface ErrorProps {
+    error: Error & { digest?: string };
+    reset: () => void;
+}
+
+export default function Error({ error, reset }: ErrorProps) {
```

Biome (2.3.13)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)